k9mail-library/src/main/java/com/fsck/k9/mail/internet/BinaryTempFileBody.java

```diff
@@ -137,12 +137,14 @@ public void close() throws IOException {
                 super.close();
             } finally {
                 Log.d(K9MailLib.LOG_TAG, "deleting temp file");
-                mFile.delete();
+                boolean fileDeleteResult = mFile.delete();
```

Comment:
My suggestion for the variable name is `fileSuccessfullyDeleted`. That makes the `if` statement below much easier to read.